src/miner.cpp

```diff
@@ -2188,6 +2188,8 @@ void static BitcoinMiner()
         if (minerThreads != NULL)
         {
             minerThreads->interrupt_all();
+            std::cout << "Waiting for mining threads to stop..." << std::endl;
```

Comment:
```suggestion
            // std::cout << "Waiting for mining threads to stop..." << std::endl;
```
We don't need extra spam in stdout ... i would recommend to comment it or output with `LogPrint` (with category) or with `LogPrintf` in `debug.log`. All of these `printf`, `fprintf(stderr`, `cout <<`, `cerr <<` in code are strange, especially in Qt part (i mean that mining code using not only by daemon, but by Qt wallet also).

Also i offer to consider idea to change all `printf`, `fprintf(stderr`, `cout <<`, `cerr <<` on `LogPrint` / `LogPrintf`, as it already implemented in Komodo-Qt (KomodoOcean).